src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="month month--sun month--31">
```

Comment:
Still not fixed from the previous review
<img width="771" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/64750958/0f9688c8-e0d2-45a0-9f77-1409942ac97e">